README.md

```diff
@@ -17,16 +19,32 @@ Mirecka J, Famili M, Kotanska A, Jurashcko N, Costa-Gomes B, Palmer CM, Thiyagal
 
 > Note: This has been tested in the `refactor` branch.
 
-You can install the libraries needed for this package on a [fresh virtual environment](https://packaging.python.org/en/latest/guides/installing-using-pip-and-virtual-environments/) with the following:
+You can install the libraries needed for this package on a
+[fresh virtual environment](https://packaging.python.org/en/latest/guides/installing-using-pip-and-virtual-environments/)
+with the following:
 
 ```
 python3 -m venv env
 source env/bin/activate
-pip install -e .
+python -m pip install -e .
```

Comment:
Also, this failed for me initially, but after some Googling turns out this will work for `pip > 21.3`, so maybe worth adding that into the installation instructions.

Reply:
ok i'll add a note on always running `python -m pip install --upgrade pip` before installing.